### PR TITLE
Add dracula look

### PR DESCRIPTION
**Pull request by jjsearle**
Add Dracula 'look'

### CODE REVIEW

File: usr/share/regolith-look/dracula/compositor-init [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+while [ ! -z "$compositor_PID" ]; do
+  kill $compositor_PID
+  sleep .1
+  compositor_PID=$(pidof picom)
+done
+
+if [[ -f "$HOME/.config/regolith2/picom/config" ]]; then
+  /usr/bin/picom --config "$HOME/.config/regolith2/picom/config"
+else
+  /usr/bin/picom --config /usr/share/regolith-look/nord/picom.conf
```
Comment (kgilmer) [inline usr/share/regolith-look/dracula/compositor-init:15]: this path is invalid, it needs to point to your version of the picom.conf.  I think it's just swapping `nord` for `dracula`.

**Review by kgilmer — CHANGES_REQUESTED** [review]
Other than the one path fix, looks great!

File: usr/share/regolith-look/dracula/compositor-init (hunk 1 above)
Comment (jjsearle) [reply usr/share/regolith-look/dracula/compositor-init:15]: Yep, you are correct. I will push a fix shortly.

Comment (jjsearle) [reply usr/share/regolith-look/dracula/compositor-init]: I have fixed the path and re-pushed.

**Review by kgilmer — APPROVED** [review]
LGTM

**Pull request merged**